### ISSUE TITLE
Issue - 500Internal Server Error trigger VR.446 and VR.209 - Environments u-002/001

### ISSUE THREAD

**Issue by IreneDataHub** [Energinet-DataHub/geh-charges#1009]
After sending the Postman messages "create subscription with to long  name" and "outside time" I got the error code 500. 

Please investigate and see if there is an issue with the messages.

![image.png](https://images.zenhubusercontent.com/612f6568004bdd03eab7ca46/735e44b5-54b3-40e2-ae73-5aec60664782)

**Tech Notes**
Seems like we are not picking up the resolution properly in code.

Error and stack trace from U-002 (application insights "Failures" blade):

_"One or more errors occurred. (Provided Resolution value 'Unknown' is invalid and cannot be mapped.) Provided Resolution value 'Unknown' is invalid and cannot be mapped."_

```
System.AggregateException:
   at System.Threading.Tasks.Task`1.GetResultCore (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Threading.Tasks.Task`1.get_Result (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at Microsoft.Azure.Functions.Worker.Invocation.DefaultFunctionInvoker`2+<>c.<InvokeAsync>b__6_0 (Microsoft.Azure.Functions.Worker.Core, Version=1.4.0.0, Culture=neutral, PublicKeyToken=551316b6919f366c: D:\a\1\s\src\DotNetWorker.Core\Invocation\DefaultFunctionInvoker.cs:32)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Threading.Tasks.Task+<>c.<.cctor>b__277_0 (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Threading.ExecutionContext.RunFromThreadPoolDispatchLoop (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at Microsoft.Azure.Functions.Worker.Invocation.DefaultFunctionExecutor+<ExecuteAsync>d__4.MoveNext (Microsoft.Azure.Functions.Worker.Core, Version=1.4.0.0, Culture=neutral, PublicKeyToken=551316b6919f366c: D:\a\1\s\src\DotNetWorker.Core\Invocation\DefaultFunctionExecutor.cs:45)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at Microsoft.Azure.Functions.Worker.OutputBindings.OutputBindingsMiddleware+<Invoke>d__0.MoveNext (Microsoft.Azure.Functions.Worker.Core, Version=1.4.0.0, Culture=neutral, PublicKeyToken=551316b6919f366c: D:\a\1\s\src\DotNetWorker.Core\OutputBindings\OutputBindingsMiddleware.cs:16)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at GreenEnergyHub.Charges.Infrastructure.Core.Function.FunctionInvocationLoggingMiddleware+<Invoke>d__2.MoveNext (GreenEnergyHub.Charges.Infrastructure.Core, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: /home/runner/work/geh-charges/geh-charges/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure.Core/Function/FunctionInvocationLoggingMiddleware.cs:40)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at GreenEnergyHub.Charges.Infrastructure.Core.MessageMetaData.MessageMetaDataMiddleware+<Invoke>d__3.MoveNext (GreenEnergyHub.Charges.Infrastructure.Core, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: /home/runner/work/geh-charges/geh-charges/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure.Core/MessageMetaData/MessageMetaDataMiddleware.cs:42)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at GreenEnergyHub.Charges.Infrastructure.Core.Correlation.FunctionTelemetryScopeMiddleware+<Invoke>d__3.MoveNext (GreenEnergyHub.Charges.Infrastructure.Core, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: /home/runner/work/geh-charges/geh-charges/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure.Core/Correlation/FunctionTelemetryScopeMiddleware.cs:58)
Inner exception System.ComponentModel.InvalidEnumArgumentException handled at System.Threading.Tasks.Task`1.GetResultCore:
   at GreenEnergyHub.Charges.Infrastructure.Core.Cim.Charges.ResolutionMapper.Map (GreenEnergyHub.Charges.Infrastructure.Core, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: /home/runner/work/geh-charges/geh-charges/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure.Core/Cim/Charges/ResolutionMapper.cs:47)
   at GreenEnergyHub.Charges.Infrastructure.CimDeserialization.ChargeBundle.ChargeCommandConverter+<ParsePointAsync>d__8.MoveNext (GreenEnergyHub.Charges.Infrastructure, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: /home/runner/work/geh-charges/geh-charges/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure/CimDeserialization/ChargeBundle/ChargeCommandConverter.cs:284)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at GreenEnergyHub.Charges.Infrastructure.CimDeserialization.ChargeBundle.ChargeCommandConverter+<ParseSeriesPeriodIntoOperationAsync>d__6.MoveNext (GreenEnergyHub.Charges.Infrastructure, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: /home/runner/work/geh-charges/geh-charges/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure/CimDeserialization/ChargeBundle/ChargeCommandConverter.cs:222)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at GreenEnergyHub.Charges.Infrastructure.CimDeserialization.ChargeBundle.ChargeCommandConverter+<ParseChargeTypeElementIntoOperationAsync>d__5.MoveNext (GreenEnergyHub.Charges.Infrastructure, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: /home/runner/work/geh-charges/geh-charges/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure/CimDeserialization/ChargeBundle/ChargeCommandConverter.cs:170)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at GreenEnergyHub.Charges.Infrastructure.CimDeserialization.ChargeBundle.ChargeCommandConverter+<ParseChargeGroupIntoOperationAsync>d__4.MoveNext (GreenEnergyHub.Charges.Infrastructure, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: /home/runner/work/geh-charges/geh-charges/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure/CimDeserialization/ChargeBundle/ChargeCommandConverter.cs:84)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at GreenEnergyHub.Charges.Infrastructure.CimDeserialization.ChargeBundle.ChargeCommandConverter+<ParseChargeOperationsAsync>d__3.MoveNext (GreenEnergyHub.Charges.Infrastructure, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: /home/runner/work/geh-charges/geh-charges/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure/CimDeserialization/ChargeBundle/ChargeCommandConverter.cs:70)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1+ConfiguredTaskAwaiter.GetResult (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at GreenEnergyHub.Charges.Infrastructure.CimDeserialization.ChargeBundle.ChargeCommandConverter+<ConvertSpecializedContentAsync>d__2.MoveNext (GreenEnergyHub.Charges.Infrastructure, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: /home/runner/work/geh-charges/geh-charges/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure/CimDeserialization/ChargeBundle/ChargeCommandConverter.cs:48)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1+ConfiguredTaskAwaiter.GetResult (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at GreenEnergyHub.Charges.Infrastructure.CimDeserialization.MarketDocument.DocumentConverter+<ConvertAsync>d__2.MoveNext (GreenEnergyHub.Charges.Infrastructure, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: /home/runner/work/geh-charges/geh-charges/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure/CimDeserialization/MarketDocument/DocumentConverter.cs:37)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1+ConfiguredTaskAwaiter.GetResult (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at GreenEnergyHub.Charges.Infrastructure.CimDeserialization.ChargeBundle.ChargeCommandDeserializer+<ConvertAsync>d__2.MoveNext (GreenEnergyHub.Charges.Infrastructure, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: /home/runner/work/geh-charges/geh-charges/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure/CimDeserialization/ChargeBundle/ChargeCommandDeserializer.cs:38)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1+ConfiguredTaskAwaiter.GetResult (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at Energinet.DataHub.Core.Messaging.Transport.SchemaValidation.SchemaValidatingMessageDeserializer`1+<FromBytesAsync>d__2.MoveNext (Energinet.DataHub.Core.Messaging, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at Energinet.DataHub.Core.Messaging.Transport.SchemaValidation.SchemaValidatingMessageDeserializer`1+<FromBytesAsync>d__2.MoveNext (Energinet.DataHub.Core.Messaging, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1+ConfiguredTaskAwaiter.GetResult (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at Energinet.DataHub.Core.Messaging.Transport.MessageExtractor+<ExtractAsync>d__2.MoveNext (Energinet.DataHub.Core.Messaging, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1+ConfiguredTaskAwaiter.GetResult (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at GreenEnergyHub.Charges.Infrastructure.Core.MessagingExtensions.ValidatingMessageExtractor`1+<ExtractAsync>d__1.MoveNext (GreenEnergyHub.Charges.Infrastructure.Core, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: /home/runner/work/geh-charges/geh-charges/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure.Core/MessagingExtensions/ValidatingMessageExtractor.cs:39)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable`1+ConfiguredTaskAwaiter.GetResult (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at GreenEnergyHub.Charges.FunctionHost.Charges.ChargeIngestion+<ValidateMessageAsync>d__5.MoveNext (GreenEnergyHub.Charges.FunctionHost, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: /home/runner/work/geh-charges/geh-charges/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/Charges/ChargeIngestion.cs:77)
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification (System.Private.CoreLib, Version=5.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)
   at GreenEnergyHub.Charges.FunctionHost.Charges.ChargeIngestion+<RunAsync>d__4.MoveNext (GreenEnergyHub.Charges.FunctionHost, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null: /home/runner/work/geh-charges/geh-charges/source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/Charges/ChargeIngestion.cs:52)
```

![image](https://user-images.githubusercontent.com/11583438/148530490-4d0b087a-0596-4f80-a259-417c1cccaec8.png)

**Comment by HenrikSommer** [comment]
@IreneDataHub and I have found that this bug is no longer present.
It has probably been resolved by [pull request #1057](https://github.com/Energinet-DataHub/geh-charges/pull/1057)

**Comment by prtandrup** [comment]
Tested✅ in U-001
Violating VR.209 "time limit" - request:

![image.png](https://images.zenhubusercontent.com/60ae42d0916bb92d8b000901/2cfea56c-9e79-4b25-821f-549124379041)

Rejection received:

![image.png](https://images.zenhubusercontent.com/60ae42d0916bb92d8b000901/6578d9ff-2a50-4e29-b323-22682cdd3de7)

Violating VR.446 "name length" - request:

![image.png](https://images.zenhubusercontent.com/60ae42d0916bb92d8b000901/055e0849-eeb6-4c4b-9921-9c8c79349ef3)

Rejection received:

![image.png](https://images.zenhubusercontent.com/60ae42d0916bb92d8b000901/5dd8e5e2-ac4f-495b-8a3c-14d597c611b2)
